azurecosmos/src/main/java/site/ycsb/db/AzureCosmosClient.java

```diff
@@ -224,6 +226,13 @@ private void initAzureCosmosClient() throws DBException {
       gatewayConnectionConfig.setIdleConnectionTimeout(Duration.ofSeconds(gatewayIdleConnectionTimeoutInSeconds));
     }
 
+    String preferredRegions = this.getStringProperty("azurecosmos.preferredRegionList", null);
+    List<String> preferredRegionList = null;
+    if (StringUtils.isNotEmpty(preferredRegions)) {
```

Comment:
If Java 11+ we can use its String.isEmpty()

Reply:
@karianna YCSB compiler plugin is target at Java 8 https://github.com/Azure/YCSB/blob/ff112196509153f20f8fa487f49c2aa5d598d13f/pom.xml#L250